### DISCUSSION TITLE
Input Fields : entering multiple items

### DISCUSSION THREAD

**Discussion by wilfredomerced** [twilio-labs/paste#1481] [Q&A] [answered]
Was curious if we have a pattern in paste for entering multiple items in an input field like shown below. Or if any other teams are working on something similar. 

In the example you can separate multiple inputs either by comma or creating a new line.

<img width="1792" alt="Screen Shot 2021-05-21 at 11 43 23 AM" src="https://user-images.githubusercontent.com/81593493/119179553-48f0cc80-ba2c-11eb-8e39-f0b1c4d2a40c.png">

**Comment by kate-porter** [comment, marked as answer]
Hey @wilfredomerced! We don't have a documented pattern for this yet, but this is definitely done a lot throughout the products.

For this, we'd recommend using either an Input or a Textarea component and providing help text to let the user know how they should input multiple values, which is what you've got in the screenshot you shared.

Here's an example of a similar pattern that's being used in Console:
![image](https://user-images.githubusercontent.com/5374045/119187053-07651f00-ba36-11eb-931e-9c9076bf3322.png)

Looks like they went with an Input instead of a Textarea in this case, since the user is limited to 5 values. I think a Textarea is probably best for something like your Unsubscribe case, since that could be a pretty long list of email addresses.

Hope that helps - let us know if you have any other questions!
